### PR TITLE
Replace Numpy arrays by torch tensors

### PR DESCRIPTION
**Pull request by ramesht007**
## Description

This pull request is related to issue #100 for Replacing Numpy arrays by torch tensors

### CODE REVIEW

File: tests/local_mode/test_language.py [hunk 1]
```diff
@@ -39,6 +40,7 @@ def test_vector_valid_token_is_not_zero():
 
     doc = nlp("banana")
     actual = doc[0].vector
+    # changed from numpy to torch zeros tensor
     zeros = np.zeros(actual.shape)
```
Comment (AlanAboudib) [inline tests/local_mode/test_language.py:44]: Please make this a torch tensor instead of a numpy array.

File: tests/local_mode/test_language.py [hunk 2]
```diff
@@ -50,6 +52,7 @@ def test_vector_non_valid_token_is_zero():
 
     doc = nlp("outofvocabularytoken")
     actual = doc[0].vector
+    # changed from numpy to torch zeros tensor
     zeros = np.zeros(actual.shape)
```
Comment (AlanAboudib) [inline tests/local_mode/test_language.py:56]: Please make this a torch tensor instead of a numpy array.

Comment (ramesht007) [reply tests/local_mode/test_language.py:56]: ok I will work on it🙂

**Comment by ramesht007** [comment]
@Nilanshrajput how should I proceed

**Comment by Nilanshrajput** [comment]
@ramesht007 I have realised as this is little complex issue for a beginner, so I am removing numpy use from various files, I will advise you to go through all the tests and check for import of bumpy remove the np use from tests.

**Comment by ramesht007** [comment]
ok I will start working on it

File: syfertext/pointers/doc_pointer.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 from syft.workers.base import BaseWorker
 import syft as sy
 
-from .span_pointer import SpanPointer
+from .span_pointer import Spaointer
```
Comment (AlanAboudib) [inline syfertext/pointers/doc_pointer.py:5]:
```suggestion
from .span_pointer import SpanPointer
```

**Comment by Nilanshrajput** [comment]
@AlanAboudib  I see you lurking around, i think the pr is now  good to merge, phew

**Comment by ramesht007** [comment]
😆

**Comment by Nilanshrajput** [comment]
Congrats @ramesht007  for your first contribution. Good Work

**Comment by Nilanshrajput** [comment]
Great work @ramesht007 , thanks for your contribution

**Comment by ramesht007** [comment]
Thank you very much @Nilanshrajput @AlanAboudib 😊

**Comment by AlanAboudib** [comment]
Congrats @ramesht007 on your first SyferText contribution :100:

**Pull request merged**